src/lib/resource/sections/whats-the-crime-reference-number.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
                 },
                 'i-dont-know-the-crime-reference': {
                     description:
-                        '{% from "components/details/macro.njk" import govukDetails %}{{ govukDetails({summaryText: "Help with your crime reference number",html: \'<p class="govuk-body">If you do not have your crime reference number, you can call 101 to speak to your local police station.</p><p class="govuk-body">If you have more than one crime reference number, you can provide additional details later in this claim.</p>\'})}}'
+                        '{% from "components/details/macro.njk" import govukDetails %}{{ govukDetails({summaryText: "Help with your crime reference number",html: \'<p class="govuk-body">If you have more than one crime reference number, you can add these numbers to the ‘additional details’ section at the end of the application.</p><p class="govuk-body">If you do not have your crime reference number, call 101 to speak to your local police station. They can help you get this.</p>\'})}}'
```

Comment:
Are those fancy single quotation marks around "additional details" going to cause any escaping issues anywhere? I only ask because they're inside a pair of escaped single quotes.

Reply:
Good question, seems okay?
![image](https://user-images.githubusercontent.com/41323244/141811162-f5519340-3bed-4903-bd57-0e61188f63c0.png)

Reply:
But is it a standard pattern/generally used?

Reply:
I've had a lot of issues with parsing those nunjucks components, but generally I just use single, double or back ticks. I'm not entirely sure how you even use those specific ones, without copy and pasting them from a text editor? But if it works, and it's compatible with different browsers, etc, then I'm happy to go with it. I wasn't requesting a change, as such, I just genuinely don't know if this causes problems anywhere.

Reply:
Tested on firefox 93.0, Chrome 95.0.4638.69 and IE 
![image](https://user-images.githubusercontent.com/41323244/142590977-ad987631-5b81-4bfa-a5f6-946cc016a99c.png)
We will just go with it.